### ISSUE TITLE
How to use Beta Init Command???

### ISSUE THREAD

**Issue by abhinavjha126** [kubeedge/kubeedge#4528]
Hello,
I am using Kubeedge Version 1.11.1 along with Kubernetes version 1.22.6

Commands Used:
wget https://github.com/kubeedge/kubeedge/releases/download/v1.11.1/keadm-v1.11.1-linux-amd64.tar.gz
tar xzvf keadm-v1.11.1-linux-amd64.tar.gz
cd keadm-v1.11.1-linux-amd64/keadm/
cp -r keadm /usr/local/bin

root@core1:~/keadm-v1.11.1-linux-amd64/keadm# keadm beta --help
keadm beta command provides some subcommands that are still in testing, but have complete functions and can be used in advance, but now it contains nothing

Using keadm beta command is showing this output, how can we use that command?????????
Please let me know, it's urgent.

**Comment by wackxu** [comment]
please use keadm command directly and do not add the beta

**Comment by abhinavjha126** [comment]
> please use keadm command directly and do not add the beta

Ok.... I will do that....But in v1.11.1 release note it's written that keadm init and join command has been deprecated and has also been advised to use keadm beta command....Any reason why it's not working?

**Comment by panpan0000** [comment]
the documents should be refined.

keadm beta init still lives in Chinese doc

`https://kubeedge.io/zh/docs/setup/keadm_zh/#keadm-beta-init`

**Comment by wackxu** [comment]
> the documents should be refined.
> 
> keadm beta init still lives in Chinese doc
> 
> `https://kubeedge.io/zh/docs/setup/keadm_zh/#keadm-beta-init`

Yes, We have some task to rewrite the doc. thanks for your suggestion
